app/controllers/coronavirus_form/afford_food_controller.rb

```diff
@@ -8,6 +8,7 @@ def submit
 
     invalid_fields = validate_radio_field(
       controller_name,
+      "getting_food",
```

Comment:
Do you think it's worth having a "group" method, a bit like "previous_path" which could be used to define the group per controller?  I think that would also allow the radio button partial to be called with:

```
<%= render partial: "question_radio_buttons", locals: {
  question: "afford_food",
  group: group,
} %>
```

Reply:
We could probably do the same for "question"

Reply:
It can be done!